### PR TITLE
Sort candidates in decreasing order

### PR DESCRIPTION
**Pull request by wbprice**
screen shot 2017-08-15 at 4 06 27 pm

### CODE REVIEW

File: frontend/js/components/environments/Results.js [hunk 1]
```diff
@@ -35,9 +45,11 @@ class Results extends Component {
             return o.matchRate == bestRate;
         });
 
-        const otherMatches = matches.filter((o) => {
-            return o.matchRate != bestRate;
-        });
+        const otherMatches = matches
+            .filter((o) => {
+                return o.matchRate != bestRate;
```
Comment (qwo) [inline frontend/js/components/environments/Results.js:50]: triple equity

File: frontend/js/components/environments/Results.js [hunk 2]
```diff
@@ -50,21 +62,23 @@ class Results extends Component {
...
-                              "It's a match!"}
-                            name={m.name}
-                            office={m.office}
-                            matchRate={Math.round(m.matchRate)} />);
+                        key={m.name}
+                        matchText={bestMatch.length > 1 ?
+                            "It's a duplicate!" :
+                            "It's a match!"}
+                        name={m.name}
+                        party="Party Affiliation"
```
Comment (qwo) [inline frontend/js/components/environments/Results.js:70]: this is static so i assume its a todo because we dont have it in the model

Comment (wbprice) [reply frontend/js/components/environments/Results.js:70]: Good catch.

File: frontend/js/components/environments/Results.js (hunk 1 above)
Comment (wbprice) [reply frontend/js/components/environments/Results.js:50]: I don't understand this comment.

File: frontend/js/components/environments/Results.js (hunk 2 above)
Comment (qwo) [reply frontend/js/components/environments/Results.js:70]: party-affiliation is dummy data? should be be pulling this from what we pass down.

Comment (wbprice) [reply frontend/js/components/environments/Results.js:70]: Party affiliation isn't part of the response from the API (yet)

**Review by ttavenner — APPROVED** [review]
Tested and approved.

**Pull request merged**